actionpack/CHANGELOG.md

```diff
@@ -6,6 +6,14 @@
 
     *Prathamesh Sonpatki*
 
+*   More explicit error message when running `rake routes`. `CONTROLLER` argument
+    can now be supplied in different ways:
+    `Rails::WelcomeController`, `Rails::Welcome`, `rails/welcome`
+
+    Fixes #22918
+
+    *Edouard Chin*
+
```

Comment:
My bad :sweat_smile:, its modifed (github didn't take the change in consideration for some reasons), commits are squashed as well. Thanks!